requirements/index.html

```diff
@@ -0,0 +1,248 @@
+						<li>
+							<strong>Governance</strong>: Utilize tools that allow interested parties to predict when issues important to them are being discussed. Maintain a backlog that reflects issues along with their status. </li>
+						<li>
+							<strong>Backwards compatibility</strong> The WCAG 3 guidelines will take a cautious approach to backwards compatibility to early versions of WCAG 3. After the first publication of WCAG 3 guidance and conformance model the guidelines aim to be backwards compatible. However, it will not be bound to backwards compatibility if a substantially more accessible outcome can be achieved. Where there are backwards-incompatible changes, the changes will be highlighted in the supplementary materials so that users of the guidelines know exactly what has changed.
+						</li>
+					</ul>
+				</section>
+			</section>
+		<section>
+		  <h2 id="design_principles">Design Principles</h2>
+          <p>The WCAG 3.0 Design Principles are based on the requirements of WCAG 2.0 and build on those requirements to meet needs identified in the WCAG 3.0 research. </p>
+          <p>Accessibility guidelines should:</p>
+          <ol>
+            <li>Support the needs of a wide range of people with disabilities and recognize that people have individual and multiple needs.</li>
+            <li>Support a measurement and conformance structure that includes guidance for a broad range of disabilities. This includes particular attention to the needs of low vision and cognitive accessibility, whose needs don't tend to fit the true/false statement success criteria of WCAG 2.x.</li>
+            <li>Be flexible enough to support the needs of people with disabilities and keep up with emerging technologies. The information structure allows guidance to be added or removed.</li>
+            <li>Be accessible and conform to the Guidelines.</li>
+            <li>Be written in plain language, as easy as possible to understand.</li>
+			<li>Improve the ability to support automated testing where appropriate and provide a procedure for repeatable tests when manual testing is appropriate.</li>
+			<li>Provide requirements to address functional needs, while avoiding or minimizing negative impacts on other functional needs. </li>
+			<li>Include accompanying documents that provide clear, concise, and findable technical information quickly in one place.</li>
+			<li>Include, where possible and appropriate, links to instruction videos, illustrations, and how-to guides. Creation of new videos and illustrations are outside the scope of the working group at this time.</li>
+			<li>Normative requirements should be unambiguous so there is only one possible meaning for each requirement.</li>
+          </ol>
+          <p>The creation process for the guidelines should:</p>
+          <ol start="7">
```

Comment:
This needs to be updated to start="11" as there were 4 items added to the list before it.